src/report-filter-controls.tsx

```diff
@@ -1,18 +1,18 @@
 import { createContext, VNode, ComponentChildren } from "preact";
-import { StateUpdater, useRef, useState } from "preact/hooks";
-import { utcFormat, utcParse } from "d3-time-format";
+import { StateUpdater, useRef, useContext } from "preact/hooks";
+import { utcFormat } from "d3-time-format";
 import { utcWeek, CountableTimeInterval } from "d3-time";
 import { route } from "./router";
 
 const yearMonthDayFormat = utcFormat("%Y-%m-%d");
-const yearMonthDayParse = utcParse("%Y-%m-%d");
 const DEFAULT_IAL = 1;
 const DEFAULT_ENV = "prod";
 
 const endOfPreviousWeek = utcWeek.floor(new Date());
 const startOfPreviousWeek = utcWeek.floor(new Date(endOfPreviousWeek.valueOf() - 1));
```

Comment:
These are now duplicated, could maybe move these and just set the defaults in the context as `new Date()` ? or `null as Date` or something nefarious?

Reply:
went ahead and simplified in 0c7f4fca4c07c1e799be097039e18b446497c8f8